src/Microsoft.Content.Build.MerfToSDPForJava.DataContracts/ViewModel/PageModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Microsoft.Content.Build.MerfToSDPForJava.DataContracts
+{
+    using System;
+    using System.Collections.Generic;
+    using YamlDotNet.Serialization;
+
+    [Serializable]
+    public class PageModel
+    {
+        [YamlMember(Alias = "items")]
+        public List<ArticleItemYaml> Items { get; set; } = new List<ArticleItemYaml>();
+
+        [YamlMember(Alias = "metadata")]
+        public Dictionary<string, object> Metadata { get; set; }
+
+        [YamlMember(Alias = "references")]
+        public List<ReferenceViewModel> References { get; set; } = new List<ReferenceViewModel>();
```

Comment:
if we do not need the information in "References", maybe it is not necessary to deserialize it.

Reply:
Removed